doc/source/science_guide/sg_dynamics.rst

```diff
@@ -734,3 +733,13 @@ Finally, as with the classic EVP approach, the stresses are initialized using th
 The revised EVP is activated by setting the namelist parameter ``revised_evp = true``. 
 In the code :math:`\alpha` is ``arlx`` and :math:`\beta` is ``brlx``. The values of ``arlx`` and ``brlx`` can be set in the namelist. 
 It is recommended to use large values of these parameters and to set :math:`\alpha=\beta` :cite:`Kimmritz15`.
+
+.. _evp1d:
+
+****************
+1d EVP solver
+****************
+
+The standard EVP solver iterates hundreds of times, where each iteration includes a communication through MPI and a limited number of calculations. This limits how much the solver can be optimized as the speed is primarily determined by the communication. The 1d EVP solver circumveines the communication by utilzing shared memory, which removes the requirement for calls to the mpi communicator. As a consequence of this the potential scalability of the code is improved. The performance is best on shared memory but the solver is also functional on MPI and hybrid MPI/OpenMP setups as it will run on the master processor alone.
+
+The scalability of geophysical models is in general terms limited by the memory usage. In order to optimize this the 1d EVP solver solves the same equations that are outlined in the section :ref:`stress-evp` but it transforms all matrices to vectors (1d matrices) as this complies better with the computer hardware. The vectorization and the contiguous placement of arrays in the memory makes it easier for the compiler to optimize the code and pass pointers instead of copying the vectors. The 1d solver is not supported for tripole grids and the code will abort if this combination is attempted.
```

Comment:
I think complies should be compiles?

as this compiles better with the computer hardware